app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim

```diff
@@ -1,6 +1,6 @@
 .TopBarLayout
   .TopBarLayout-header
-    md-toolbar.Toolbar.Toolbar--clean
+    md-toolbar.md-whiteframe-z1.Toolbar.Toolbar--clean
```

Comment:
There is a shadow https://trello.com/c/Sj9eGWbf/37-fix-textarea-input-on-new-commnet-screen